core/blockchain_action.go

```diff
@@ -441,10 +441,14 @@ func (chain *FullBlockChain) executeTransaction(block *types.Block) (bool, *exec
 
 	statehash, evitTxs, transactions, receipts, gasFee, err := chain.executor.Execute(state, block.Header, block.Transactions, false, nil)
 	txTree := calcTxTree(transactions)
-	if txTree != block.Header.TxTree{
+	if txTree != block.Header.TxTree {
 		Logger.Errorf("Fail to verify txTree, hash1:%s hash2:%s", txTree, block.Header.TxTree)
 		return false, nil
 	}
+	if len(transactions) != len(block.Transactions) {
```

Comment:
no need to check this, because the txTree check already work

Reply:
explained